src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/InputLambdaBodyLengthDefault.java

```diff
@@ -0,0 +1,100 @@
+package com.puppycrawl.tools.checkstyle.checks.sizes.lambdabodylength;
+
+import java.util.List;
+import java.util.function.*;
+
+/*
+* Config = default
+ */
+public class InputLambdaBodyLengthDefault {
+    {
+        List<String> list = null;
+        list.forEach(item -> { // violation
+            System.out.println(1);
+            System.out.println(2);
+            System.out.println(3);
+            System.out.println(4);
+            System.out.println(5);
+            System.out.println(6);
+            System.out.println(7);
+            System.out.println(8);
+            System.out.println(9);
+            System.out.println(10);
+        });
+        list.forEach(item -> System.out.println( // violation
+            item.trim()
+                .trim()
```

Comment:
please add line number in argument here too.

Please do this for all cases where applicable.

Reply:
done